File: JavaScript/Array 2D y 3D/Orden por filas de matriz 2D.js
const matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

console.log("---Recorrido por filas---");
for (let i = 0; i < matriz.length; i++) {
    let fila = "Fila " + i + ": ";
    for (let j = 0; j < matriz[i].length; j++) {
        fila += matriz[i][j] + " ";
    }
    console.log(fila);
}

File: Python/Array 2D y 3D/Orden por filas de matriz 2D.py
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

print("---Recorrido por filas---")
for i in range(len(matriz)):
    print("Fila", i, ":", end=" ")
    for j in range(len(matriz[i])):
        print(matriz[i][j], end=" ")
    print()